README.md

```diff
@@ -1 +1,114 @@
 # OKRs 2018 Q4
+
+## Bring 'Code as Data' to market with enterprises
+
+### Ensure gitbase interoperability with 3rd party tools [DR]
+
+- [ ] Drivers (JS, PHP, Python, Ruby, Go, Java, .NET)
```

Comment:
that points are in order of priority?

Reply:
Not exactly. We could break them in two tiers for priorities:
- Drivers for in-house languages (Python, Go, Java).
- mysqldump
- grafana
- Tableau
- Drivers for other popular languages (JavaScript, Ruby, .NET, PHP).